example/integration_test/cardano_ledger_address_derivation_tests.dart

```diff
@@ -42,8 +50,16 @@ void main() {
       for (var testCase in byronTestCases) {
         test(testCase.testName, () async {
           if (isAppXS == false) {
-            final result = await deriveAddress(cardanoApp, device, testCase.network, testCase.addressParams);
-            expect(addressHexToBase58(result), equals(testCase.expectedResult));
+            try {
+              final result = await deriveAddress(cardanoApp, device, testCase.network, testCase.addressParams);
+              expect(addressHexToBase58(result), equals(testCase.expectedResult));
+            } catch (e) {
+              if (e is LedgerException) {
+                print('LedgerException caught: ${e.message}');
+              } else {
+                rethrow;
+              }
```

Comment:
rethrow here should not be inside an `else` block.

Reply:
done